src/map/latent_effect.h

```diff
@@ -36,6 +36,7 @@ enum LATENT
     LATENT_HP_OVER_VISIBLE_GEAR     = 46, //hp more than or equal to %, calculated using HP bonuses from visible gear only 
     LATENT_MP_UNDER_PERCENT         = 4,  //mp less than or equal to % - PARAM: MP PERCENT
     LATENT_MP_UNDER                 = 5,  //mp less than # - PARAM: MP #
+    LATENT_MP_OVER                  = 49, //mp greater than # - PARAM: MP #

```

Comment:
I'm temped to refactor all the IDs here and in the SQL after your PR gets merged to keep both the order it has while putting the IDs in order to prevent future mishaps.